handler/src/main/java/io/netty/handler/ssl/OpenSslEngine.java

```diff
@@ -1294,6 +1266,11 @@ private HandshakeStatus handshake() throws SSLException {
         // Adding the OpenSslEngine to the OpenSslEngineMap so it can be used in the AbstractCertificateVerifier.
         engineMap.add(this);
 
+        if (!certificateSet && keyMaterialManager != null) {
+            certificateSet = true;
+            keyMaterialManager.setServerKeyMaterial(this);
```

Comment:
Yes I see this is how the `keyMaterialManager()` method is implemented. What I'm wondering is does it add value to also have `Server/Client` in the method names. Is the usage of the `keyMaterialManager` not enough to derive context?